docs/source/index.rst

```diff
@@ -11,13 +15,14 @@ to launch experiments and analyse their data. They cover the basics of installin
 and setting up Dallinger, as well as use of the command line tools.
 
 .. toctree::
+    :caption: User Documentation
```

Comment:
I feel like this could use some more sub-categorization (e.g. installing vs using), but it's still a big improvement.